### ISSUE TITLE
Please check validity of the block starting from line #3  Fatal

### ISSUE THREAD

**Issue by mikeerickson** [sasstools/sass-lint#1010]
What version of Sass Lint are you using?

1.10.2

Please include any relevant parts of your configuration

`./node_modules/.bin/sass-lint "./resources/assets/sass/*.sass" -q -v`

And my .sass file contains the follow

```
// Fonts
@import url(https://fonts.googleapis.com/css?family=Raleway:300,400,600);
```

What did you do? Please include the actual source code causing the issue.

Executing the above command produces 

```
./resources/assets/sass/app.sass
  3:1  error  Please check validity of the block starting from line #3  Fatal
```

What did you expect to happen?

Should not throw associated error

What actually happened? Please include any error messages given to you by Sass Lint.

See above

Note:  I have reviewed other topics on this similar subject and it leads to a potential issue with `gonzales-pe`   However, I believe I am using the latest (I am using v3.4.7)

**Comment by alexdevero** [comment]
I experienced the same issue with:
```
h1 + h2,
h1 + .h2,
.h1 + h2,
.h1 + .h2, {
  margin-top: 26px;
}
```
Running sass-int produces:
```
src\scss\_base\_typography.scss
83:1  error  Please check validity of the block starting from line #83  Fatal
```

**Comment by Alexintosh** [comment]
I have the same problem with this block
```
// sass-lint:disable-all
@mixin clearfix {
    
    *zoom: 1;
    
    &::before,
    &::after {
        display: table;
        content: '';
        line-height: 0;
    }

    &::after {
        clear: both;
    }
}
// sass-lint:enable-all
```
Disabling doesn't help

**Comment by DanPurdy** [comment]
It's because it's an issue with the AST we use Gonzales-pe... Sass-lint disabling comes AFTER the AST parses your .sass & .scss files therefore disabling sass-lint rules will have no effect.

Please make separate issues here so we can track them a little easier as in the cases above they're all different reasons for failure.

Also adding an issues to the [gonzales-pe repo]() with a link to your issue here would be really helpful.

Thanks.

**Comment by Alexintosh** [comment]
Hey @DanPurdy thanks for the quick reply. 
I'm not very familiar with how AST works on Gonzales-pe, if you feel like giving more details I'll be happy to open an issue to the gonzales-pe repo. Otherwise I'll pass to ball to someone who actually knows what's going on.

**Comment by DanPurdy** [comment]
@mikeerickson & @Alexintosh both of your issues are resolved in the upcoming Gonzales-pe update so you will no longer see these fatal issues.
@alexdevero yours is still failing it seems. Will check back soon.

**Comment by alexdevero** [comment]
@DanPurdy thanks.

**Comment by JJ810** [comment]
Is there any update with this issue? I am getting the exact same errors.

**Comment by hansiemithun** [comment]
Happy 5th anniversary for this issue
